plugins/techdocs-backend/src/service/router.test.ts

```diff
@@ -160,120 +154,6 @@ describe('createRouter', () => {
   });
 
   describe('GET /sync/:namespace/:kind/:name', () => {
-    describe('accept application/json', () => {
-      it('should return not found if entity is not found', async () => {
```

Comment:
Hmm....  Are these tests that are still testing valid logic, but they just need to be updated to use event streaming instead of standard HTTP requests?